app/assets/javascripts/app/utils/countdown-timer.js

```diff
@@ -11,7 +10,6 @@ export default (targetSelector, timeLeft = 0, interval = 1000) => {
     countdownTarget.innerHTML = msFormatter(remaining);
 
     if (remaining <= 0) {
```

Comment:
Removing this means that the 'account locked' screen will no longer redirect the user back to the login screen when the timeout expires. If we are ok with this behavior, this looks good to me.